test/models/user_test.rb

```diff
@@ -320,6 +320,68 @@ def setup_user(operation)
     assert_includes record.errors.keys, :admin
   end
 
+  test "audit of password change should be saved only once, second time audited changes should not contain password_changed" do
+    user = FactoryGirl.create(:user)
```

Comment:
Please move the user creation to a `setup` block, it's the same in all 3 hosts. You can group them in a context `audits for password change` or similar.